TypeApprovalRegister+forwardings.yaml

```diff
@@ -127,6 +128,7 @@ forwardings:
       fc-port-deletion:
       operation-client-deletion:
     initiating-requests:
+      # [all oam requests]
```

Comment:
Thanks for adding comments here. Also, a comment like [all oam put requests] can be added to "initiating-requests" of OamRequestCausesLtpUpdateRequest

Reply:
I would like to proceed now, but if you would wish you could add such comment in one of the next process steps.